NAMESPACE

```diff
@@ -1,19 +1,29 @@
 # Generated by roxygen2: do not edit by hand
 
 export("%>%")
+export(h1_tabstop)
+export(h2_tabstop)
+export(h3_tabstop)
+export(h4_tabstop)
+export(h5_tabstop)
+export(h6_tabstop)
+export(nhs_card)
+export(nhs_card_tabstop)
+export(nhs_footer)
+export(nhs_grid_2_col)
+export(nhs_grid_3_col)
+export(nhs_grid_4_col)
+export(nhs_header)
+export(nhs_navlistPanel)
+export(nhs_selectInput)
+export(nhs_selectizeInput)
 export(run_app)
-export(theme_nhsbsa)
 import(shiny)
 importFrom(golem,activate_js)
 importFrom(golem,add_resource_path)
 importFrom(golem,bundle_resources)
 importFrom(golem,favicon)
 importFrom(golem,with_golem_options)
 importFrom(magrittr,"%>%")
-importFrom(shiny,HTML)
```

Comment:
Since `shiny` is imported in whole, no need for any individual imports of `shiny` functions.